### PR TITLE
docs: update group name and corresponding output folder names

### PR DESCRIPTION
**Pull request by munishm**
**Reason for Change**:
More accurate with the names even though hard coded. Would be easy to understand the example and relate. 

**Issue Fixed**:
#1258 

**Requirements**:

- [ ] uses [conventional commit messages](https://www.conventionalcommits.org/)
  
- [ ] includes documentation
- [ ] adds unit tests
- [ ] tested upgrade from previous version

**Notes**:

### CODE REVIEW

File: docs/topics/windows.md [hunk 1]
```diff
@@ -385,30 +385,32 @@ It will also create a working Kubernetes client config file in `_output/<dnspref
...
 
 ```json
       "masterFQDN": {
         "type": "String",
-        "value": "plangk8swin1.westus2.cloudapp.azure.com"
+        "value": "wink8s1.westus2.cloudapp.azure.com"
       },
 ```
 
+A dnsPrefix which forms part of the the hostname for your cluster (e.g. staging, prodwest, blueberry). The DNS prefix must be unique so pick a random name. In case you have not modified the dns prefix, it might error out.
```
Comment (mboersma) [inline docs/topics/windows.md]: This isn't a full sentence, and I'm not sure what it relates to. Is this necessary here?

Comment (munishm) [reply docs/topics/windows.md]: DnsPrefix has to be unique as this is global resource and can get into errors if some other user has it under his/her name.
It is more about letting the user know to make it unique.  Wasn't sure how to add this so took a cue from the other documentation.

Comment (CecileRobertMichon) [reply docs/topics/windows.md]:
```suggestion
The DNS prefix provided is used to generate the hostname of the cluster (e.g. staging, prodwest, blueberry), and must be unique for each cluster deployment.
```

Comment (CecileRobertMichon) [reply docs/topics/windows.md]: @mboersma thoughts?

Comment (mboersma) [reply docs/topics/windows.md]: That fixes it. 👍

**Comment by mboersma** [comment]
@PatrickLang could you take a look at this Windows doc change?

File: docs/topics/windows.md (hunk 1 above)
Comment (mboersma) [reply docs/topics/windows.md]: @munishm if you're ok with Cécile's change above, we're ok with this PR. Thanks again!

**Pull request merged**